### PR TITLE
Better review DB

### PR DESCRIPTION
**Pull request by slawosz**
Instead of creating new database on every pull request and using seed data, it copies existing template db.

### CODE REVIEW

File: cosmetics-web/deploy-review.sh [hunk 1]
```diff
@@ -25,7 +25,9 @@ if [ -z "$DB_NAME" ]
 then
   DB_NAME=cosmetics-review-database
 fi
-cf create-service postgres small-13 $DB_NAME -c '{"enable_extensions": ["pgcrypto"]}'
+
+DB_GUID=$(cf service cosmetics-db-staging-dump-26092022 --guid)
+cf create-service postgres small-13 $DB_NAME -c "{\"restore_from_point_in_time_of\": \"$DB_GUID\" }"
```
Comment (slorek) [inline cosmetics-web/deploy-review.sh:30]: These snapshots are only kept by PaaS for 7 days, so this will stop working or need constantly updating.

Comment (slawosz) [reply cosmetics-web/deploy-review.sh:30]: For my understanding, it does creates a new DB from existing DB, I am not providing any timestamp there.

Comment (slorek) [reply cosmetics-web/deploy-review.sh:30]: I understand now, we will need to keep this `cosmetics-db-staging-dump-26092022` as a live PostgreSQL instance.

Comment (slawosz) [reply cosmetics-web/deploy-review.sh:30]: Yes @slorek I will take a fresh dump of staging and I will rename it accordingly. I will have to write less dry documentation.

File: cosmetics-web/docs/REVIEW_DB.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+To dump staging db:
```
Comment (slawosz) [inline cosmetics-web/docs/REVIEW_DB.md:1]: This require updating.

File: cosmetics-web/docs/REVIEW_APPS.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Dump enhanced db on localhost again:
+
+```
+pg_dump 'postgres://localhost/cosmetics_db?user=postgres' --file staging_enhanced_db.sql --no-acl --no-owner
+```
+
+And create db on review app:
+
+```
+cf conduit cosmetics-database-template -- psql < staging_enhanced_db.sql
```
Comment (scruti) [inline cosmetics-web/docs/REVIEW_APPS.md:93]: If I understood right, the name for the staging template `cosmetics-database-template` doesn't match the name of the template [mentioned above](https://github.com/OfficeForProductSafetyAndStandards/cosmetic-product-notifications/pull/2620/files#diff-c6146f0122beefaf82a9d940c804190251fdd3c56049ff96f93e9a2368609c79R51) `cosmetics-db-staging-template`

File: cosmetics-web/lib/tasks/opensearch.rake [hunk 4]
```diff
@@ -1,6 +1,6 @@
 namespace :open_search do
   desc "Reindex Open Search"
   task reindex: :environment do
-    ReindexOpensearchJob.perform_now
+    ReindexOpensearchJob.perform_later
```
Comment (scruti) [inline cosmetics-web/lib/tasks/opensearch.rake:4]: :question:

File: cosmetics-web/deploy-review.sh [hunk 5]
```diff
@@ -25,7 +25,9 @@ if [ -z "$DB_NAME" ]
 then
   DB_NAME=cosmetics-review-database
 fi
-cf create-service postgres small-13 $DB_NAME -c '{"enable_extensions": ["pgcrypto"]}'
+
+DB_GUID=$(cf service cosmetics-database-template --guid)
+cf create-service postgres small-13 $DB_NAME -c "{\"restore_from_point_in_time_of\": \"$DB_GUID\" }"
```
Comment (scruti) [inline cosmetics-web/deploy-review.sh:30]: :clap:

**Review by scruti — APPROVED** [review]

File: cosmetics-web/lib/tasks/opensearch.rake (hunk 4 above)
Comment (slawosz) [reply cosmetics-web/lib/tasks/opensearch.rake:4]: Yes, it looks like it doesn't work anyway. One thing to investigate, for now giving it a try.

**Pull request merged**